cartographer_ros/cartographer_ros/sensor_bridge.cc

```diff
@@ -100,7 +100,17 @@ void SensorBridge::HandleNavSatFixMessage(
 void SensorBridge::HandleLandmarkMessage(
     const std::string& sensor_id,
     const cartographer_ros_msgs::LandmarkList::ConstPtr& msg) {
-  trajectory_builder_->AddSensorData(sensor_id, ToLandmarkData(*msg));
+  auto landmark_data = ToLandmarkData(*msg);
+
+  auto tracking_from_camera = tf_bridge_.LookupToTracking(
```

Comment:
rename: camera -> landmark_sensor or the like

Reply:
done